docs/stories/icon.stories.js

```diff
@@ -173,13 +175,20 @@ storiesOf('API|Icon', module)
       <Showcase variant="squared">
         <AmbulanceIcon circle circleColor={palette.mediumBeetrootPurple} color="white" />
       </Showcase>
-      {[EuroIcon, ForbiddenIcon, FlatrateSt1Icon, FlatrateSt2Icon, FlatrateSt3Icon, FlatrateTsIcon, FlatrateTdIcon].map(
-        (Icon, index) => (
-          <Showcase variant="squared" key={index}>
-            <Icon circle circleColor={palette.mediumBeetrootPurple} color="white" />
-          </Showcase>
-        )
-      )}
+      {[
+        EuroIcon,
+        InvoiceIcon,
```

Comment:
I don't think it is relevant

Reply:
EuroIcon used in invoicesCard , and InvoiceIcon used in the menuDrawer as discussed earlier.